### DISCUSSION TITLE
매크로 새로고침 주기 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by TRV449** [changdoc/naver-vaccine-macro#63] [Q&A]
혹시 새로고침 주기 빠르게 했을 때 대략 몇번만에 차단이 되는걸까요?
만약 그게 정해져 있다고 하면 ex)10번
9번을 빠른 주기로 하고 텀을 가지고 다시 9번 빠르게 하고
하는 식으로는 안되겠죠?

**Comment by OneTop4458** [comment]
차단 관련 주기를 정확하게 파악하는 것은 불가능합니다
서버 단에서 차단하기 때문

노가다를 통해 파악하는 것이 유일한 방법인데 cost가 너무 많이 들어 시도하실 분이 없을 듯하네요.

파악된 바로는 1개 병원 2000ms , 2개 병원 4000ms 의 경우 차단 당하지 않았습니다.
그리고 특정 x초 x회 이상 요청 시 처럼 딱 정해진 값을 기준으로 차단 된다기 보다는

약간 방화벽처럼 스스로 일정 수준을 넘으면 자동 차단 되는 느낌이 강하더라구요..

참고로 수동으로 새고고침을 약 25회 정도 반복하면 차단 되나 매크로 1000ms의 경우 5초도 안걸리고 바로 차단 됩니다..
